demo/app.js

```diff
@@ -961,6 +991,18 @@ const Demo = React.createClass({
         />
         <br/><br/>
         <SearchInput />
+        <DisplayInput
+          hint='Click to Edit'
+          label='Display Input'
+          onBlur={this._handleInputStatusMessage}
+          onFocus={this._handleInputFocus}
+          onMouseOut={this._handleInputHideHint}
+          onMouseOver={this._handleInputShowHint}
```

Comment:
@tumentumurchudur do you want to take a look at this PR when you have a sec?  You always see more than I do and maybe you might have an answer around using `onMouseOut` and `onMouseOver` for showing/hiding the hint.